src/pages/solution/places/release-notes.md

```diff
@@ -8,6 +8,12 @@ keywords:
 
 # Release notes
 
+## August 19, 2024
+
+### Android Places 3.0.1
+
+* Removed `string.xml` file with reserved resource key `app_name` which could cause app name to be changed

```

Comment:
@spoorthipujariadobe There is a missing period.

nit: I also prefer updating the description to match assurance. 
`* Removed unused `app_name` string resource to prevent conflicts with resources defined by the app. `

Could you please update the release tag for “places” and re-run the GitHub action?